### ISSUE TITLE
Support CD Player

### ISSUE THREAD

**Issue by olni** [CDrummond/lms-material#73]
..show the "Extras" Menu, like in the standard skin.
Because the CD-Player plugin uses this folder for playing Audio-CDs through LMS.

Thx,
Oliver

**Comment by CDrummond** [comment]
Not sure how this menu is populated - I can't see any reference to it in the documentation. Also, SqueezeCtr and the default webUI show different items on my system.

I'm not sure its worth supporting this menu, the items that are on the default web UI are not what I would want to see here. And things like the SQL Playlist in extras is jsut way too complicated.

**Comment by olni** [comment]
.. i see this also as not "high priority" but nice to have. For now i use a workaround via the "play url" feature and type cdplay://<track #> but someone has to know first which track is the right one :-(
But thank you for this very good skin, which makes all the special controller apps nearly obsolete.
Oliver

**Comment by olni** [comment]
..thank you very much! This works very well - even the database query from
the plugin works like expected. See the screenshots. Well done!
😀

Oliver

Am Do., 7. Feb. 2019 um 23:19 Uhr schrieb CraigD <notifications@github.com>:

**Comment by CDrummond** [comment]
Screenshots? I guess you forgot to attach those.
I've changed the issue title to reflect the actual change - don't think I'm going to support "Extra" (not until I see something that should be there in Material). So, closing.

**Comment by olni** [comment]
...hmm, seems like my fault ..

Am Fr., 8. Feb. 2019 um 20:36 Uhr schrieb CraigD <notifications@github.com>:

**Comment by olni** [comment]
..sorry, my fault.

![cdplayer_1](https://user-images.githubusercontent.com/10218872/52503132-e40b6280-2be4-11e9-88a6-bfe5c10fee8d.png)
![cdplayer_2](https://user-images.githubusercontent.com/10218872/52503137-e66dbc80-2be4-11e9-8a99-3039be286ae4.png)
![cdplayer_3](https://user-images.githubusercontent.com/10218872/52503140-e8378000-2be4-11e9-8cf2-a5975454ff99.png)
